LabManagement/PowerShell/BulkOperations/Examples/Send-Invitations.ps1

```diff
@@ -0,0 +1,40 @@
+[CmdletBinding()]
+param(
+    [parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $CsvConfigFile,
+
+    [parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $CsvOutputFile,
+
+    [parameter(Mandatory = $false, ValueFromPipelineByPropertyName = $true)]
+    [int] $ThrottleLimit = 10
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+
+# Make sure the input file does exist
+if (-not (Test-Path -Path $CsvConfigFile)) {
+    Write-Error "Input CSV File must exist, please choose a valid file location..."
+}
+
+# Make sure the output file doesn't exist
+if (Test-Path -Path $CsvOutputFile) {
+    Write-Error "Output File cannot already exist, please choose a location to create a new output file..."
+}
+
+Import-Module ../Az.LabServices.BulkOperations.psm1 -Force
+
+$scriptstartTime = Get-Date
+Write-Host "Executing Script to send all invitations, starting at $scriptstartTime" -ForegroundColor Green
+
+$labs = $CsvConfigFile | Import-LabsCsv
+
+$labs = $CsvConfigFile | Import-LabsCsv | Send-AzLabsInvitationBulk -ThrottleLimit $ThrottleLimit
```

Comment:
This seems duplicate from line 33?  Shouldn't need to import the CSV file twice

Reply:
My mistake - i had that line in for testing and forgot to remove.  Thanks!